### DISCUSSION TITLE
Schedule and Email Reports

### DISCUSSION THREAD

**Discussion by fopashelter** [sheltermanager/asm3#1470] [Q&A / Help] [answered]
Is there a way to set up a schedule for specific reports to run and be sent via email?  I thought i saw a screen with the setup, but i can't find it now!

thanks in advance!

**Comment by Tetzor** [comment, marked as answer]
If you go to Settings->Reports and click on the report name, you will find the options there to add email addresses and set the schedule.

Jon

**Comment by fopashelter** [comment]
Perfect!  Thanks!Sent from my iPhoneOn May 17, 2024, at 3:30 AM, Jon ***@***.***> wrote:﻿
Closed #1470 as resolved.
